src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    private static String command;
+    private static String description;
+    private static Task task;
+    private static boolean isTask;
+    private static boolean isCommand;
+    private static final TaskList list = new TaskList();
+
+
     public static void main(String[] args) {
+
+        welcomeMessage();
+
+        while (true) {
+            String userInput = askForInput();
+            checkTypeOfInstruction(userInput);
+            splitInput(userInput);
+
+            if (isCommand) {
+                executeCommand();
+            } else if (isTask) {
+                createTask();
+                list.addToList(task);
+            }
+
+            if (command.equals("list") || command.equals("done")) {
+                continue;
+            }
+            printSingleTask(list.getTaskCount()-1);
+            printNoOfTasks();
+        }
+    }
+
+    public static void welcomeMessage() {
```

Comment:
Could this be renamed to printWelcomeMessage? Just to make it extra clear it is to print a message.